File: 8. Remove brackets from an algebraic expression/code.java
import java.util.Scanner;

public class Main {
    public static String solve(String s) {
        StringBuilder res = new StringBuilder();
        for (char c : s.toCharArray()) {
            if (c == '(' || c == ')') {
                continue;
            }
            res.append(c);
        }
        return res.toString();
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter a string: ");
        String inputStr = scanner.nextLine();
        System.out.println("Result after removing parentheses: " + solve(inputStr));
    }
}

File: 8. Remove brackets from an algebraic expression/code.cpp
#include <iostream>
#include <string>
using namespace std;

string solve(string s) {
    string res = "";
    for (char c : s) {
        if (c == '(' || c == ')') {
            continue;
        }
        res += c;
    }
    return res;
}

int main() {
    string input_str;
    cout << "Enter a string: ";
    getline(cin, input_str);
    cout << "Result after removing parentheses: " << solve(input_str) << endl;
    return 0;
}
